openapi/generator.go

```diff
@@ -250,6 +250,7 @@ func (g *Generator) AddOperation(path, method, tag string, in, out reflect.Type,
 		op.Description = info.Description
 		op.Deprecated = info.Deprecated
 		op.Responses = make(Responses)
+		op.XCodeSamples = info.XCodeSamples
```

Comment:
Forgot to add `XCodeSamples` field to `OperationInfo` type ?
Also, I think it requires a "helper" method in `fizz` package to set this field during the declaration of a route, otherwise, modifying the spec after the routes declaration will be difficult.

Reply:
I  ight be blind, didn't see it the first time.

Reply:
No worries 😉